static/js/textMarkIconsPosition.js

```diff
@@ -17,6 +17,7 @@ var textMarkIconsPosition = function(config) {
   this.textMarkClass = config.textMarkClass;
   this.textkMarkPrefix = config.textkMarkPrefix;
   this.adjustTopOf = config.adjustTopOf;
+  this.afterUpdateIconsPosition = config.afterUpdateIconsPosition;
```

Comment:
this.afterUpdateIconsPosition = config.afterUpdateIconsPosition || doNothing();
isso evita a condição lá de baixo. Se n está definida a gente chama o doNothing() ou function(){}